pkg/sqlite/load.go

```diff
@@ -21,25 +21,25 @@ func NewSQLLiteLoader(outFilename string) (*SQLLoader, error) {
 	}
 
 	createTable := `
-	CREATE TABLE operatorbundle (
+	CREATE TABLE IF NOT EXISTS operatorbundle (
```

Comment:
I was seeing some issues after running `make unit` multiple times. An error citing X already exists prompted this quick-fix.